components/InputField/styles.ts

```diff
@@ -7,6 +7,7 @@ const styles = StyleSheet.create({
     borderRadius: 25,
     borderWidth: 1,
     height: 50,
```

Comment:
the input should have a height of 40, because now it is taller than the button next to it in the home screen